### DISCUSSION TITLE
How to handle straggler read in Trino

### DISCUSSION THREAD

**Discussion by Smith-Cruise** [trinodb/trino#17962] [Q&A]
Just like the Hadoop client's hedged read, is there any way in Trino to solve straggler read problems?

**Comment by jerryleooo** [comment]
Related issue: https://github.com/prestodb/presto/issues/10010

Related Slack discussion: https://app.slack.com/client/TFKPX15NC/CP1MUNEUX

**Reply by hackeryang** [reply]
The updated Slack address: https://trinodb.slack.com/archives/CP1MUNEUX/p1695713019040899
